src/HoldRequestResultConsumerListener.php

```diff
@@ -144,33 +144,33 @@ protected function getItem($holdRequest)
     }
 
     /**
-     * @param Item $item
-     * @param HoldRequestResult $holdRequestResult
-     * @return null|Bib
-     * @throws APIException
+     * @param $item
+     * @param $holdRequestResult
+     * @return array|null
+     * @throws NonRetryableException
      */
-    protected function getBib($item, $holdRequestResult)
+    protected function getBibs($item, $holdRequestResult)
```

Comment:
Looks fine. Perhaps you should add type hinting here and on the sendEmail method. Is there a reason the @param definitions changed?

Reply:
👍

Reply:
Hmmm... did you add the type hinting in the method parameters that @gkallenberg mentioned?

Reply:
@kfriedman I need to re-commit the code first thing Monday, it's not showing up on GitHub.